src/app/issues/page.tsx

```
@@ -0,0 +1,147 @@
+          <option value="good first issue">Good First Issue</option>
+          <option value="beginner">Beginner</option>
+          <option value="intermediate">Intermediate</option>
+          <option value="advanced">Advanced</option>
+        </select>
+
+        <select
+          value={sdk}
+          onChange={(e) => {setSdk(e.target.value)}}
+          className="p-2 rounded border"
+        >
+          <option value="">All Repo's</option>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)